ui/src/message_popup/playtest/danson_experience_page.jsx

```diff
@@ -0,0 +1,344 @@
+/* @flow weak */
+import _ from 'lodash';
+import React from 'react';
+import VelocityTransitionGroup from "velocity-react/velocity-transition-group";
+import 'velocity-animate/velocity.ui';
+import uuid from 'uuid';
+
+import RaisedButton from 'material-ui/RaisedButton';
+import Divider from 'material-ui/Divider';
+import LinearProgress from 'material-ui/LinearProgress';
+import Snackbar from 'material-ui/Snackbar';
+import MenuItem from 'material-ui/MenuItem';
+import TextField from 'material-ui/TextField';
+
+import PopupQuestion from '../popup_question.jsx';
+import type {ResponseT} from '../popup_question.jsx';
+
+import RefreshIcon from 'material-ui/svg-icons/navigation/refresh';
+import ResponsiveFrame from '../../components/responsive_frame.jsx';
+
+import {withStudents} from '../transformations.jsx';
+
+import * as Api from '../../helpers/api.js';
+import NavigationAppBar from '../../components/navigation_app_bar.jsx';
+
+import MobileInterface from '../mobile_prototype/mobile_interface.jsx';
+import {dansonQuestions} from '../questions.js';
+
+/*
+Shows the MessagePopup game
+*/
+export default React.createClass({
+  displayName: 'MessagePopupExperiencePage',
+
+  propTypes: {
+    query: React.PropTypes.object.isRequired,
+  },
+
+  contextTypes: {
+    auth: React.PropTypes.object.isRequired
+  },
+
+  getInitialState() {
+    const isSolutionMode = _.has(this.props.query, 'solution');
+    return {
+      scaffolding: {
+        helpType: isSolutionMode ? 'none' : 'feedback',
+        shouldShowSummary: !isSolutionMode,
+        hideTimer: true,
+      },
+      gameSession: null,
+      toastRevision: false,
+      limitMs: 90000,
+      email: this.context.auth.userProfile.email
```

Comment:
This changed since the last playtest for some public demos - it's set to `unknown@mit.edu` by default (in the `AuthContainer` component).

You might want to update this for the playtest so that `email` is an empty string if `this.context.auth.userProfile.email` matches `unknown@mit.edu` to make sure the email addresses that are collected are valid.  We can talk out other ways to handle it tomorrow if you want too, it doesn't have to hold up this pull request.